bin/MangoHud.conf

```diff
@@ -143,32 +166,28 @@ background_alpha=0.5
 # frametime_color=00FF00
 # background_color=020202
 # media_player_color=FFFFFF
-# wine_color=732010
-
-### Show media player metadata
-# media_player
-# media_player_name = spotify
-# media_player_order = title,artist,album
+# wine_color=EB5B5B
+# battery_color=FF9078
 
-### Specify gpu with pci bus id for amdgpu and NVML stats.
+### Specify GPU with PCI bus ID for AMDGPU and NVML stats
 ### Set to 'domain:bus:slot.function'
-# pci_dev = 0:0a:0.0
+# pci_dev=0:0a:0.0
 
 ### Blacklist
-# blacklist =
+# blacklist=
 
-################## WORKAROUNDS #################
-### Options starting with "gl_*" are for OpenGL.
-### Specify what to use for getting display size. Options are "viewport", "scissorbox" or disabled. Defaults to using glXQueryDrawable.
-# gl_size_query = viewport
+################ WORKAROUNDS #################
+### Options starting with "gl_*" are for OpenGL
+### Specify what to use for getting display size. Options are "viewport", "scissorbox" or disabled. Defaults to using glXQueryDrawable
+# gl_size_query=viewport
 
-### (Re)bind given framebuffer before MangoHud gets drawn. Helps with Crusader Kings III.
-# gl_bind_framebuffer = 0
+### (Re)bind given framebuffer before MangoHud gets drawn. Helps with Crusader Kings III
+# gl_bind_framebuffer=0
 
-### Don't swap origin if using GL_UPPER_LEFT. Helps with Ryujinx.
-# gl_dont_flip = 1
+### Don't swap origin if using GL_UPPER_LEFT. Helps with Ryujinx
+# gl_dont_flip=1
```

Comment:
Spaces don't cost extra though

Reply:
I don't mind having extra space as long as the style is uniform, I think there was something that made me decide on having them removed, but I don't know what exactly. Do you want me to change them all to having spaces?

Reply:
Eh, let it be